docs/ipr/kb_matches.md

```diff
@@ -0,0 +1,21 @@
+# KB Matches
+
+The knowledge base (KB) matches section of the report is generated by the IPR python adapter during creation of a report. Any input core variants are annotated by matching against a GraphKB API instance.
+
+## Known vs Observed Variants
+
+As GraphKB is able to support matching non-specfic variant notation. A distinction is needed between the sample variant and the variant it was matched to by the knowledge base.
```

Comment:
This seems like it should be one sentence - ie 'non-specific variant notation, a distinction' instead of 'non-specific variant notation. A distinction'